src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseTaskDataObject.cs

```diff
@@ -57,6 +57,8 @@ public interface IRestoreDatabaseTaskDataObject
         string DefaultTargetDbName { get; }
         string TargetDatabaseName { get; set; }
 
+        bool CanDropExistingConnections { get; }
```

Comment:
>CanDropExistingConnections [](start = 13, length = 26)

I'm a little confused why this is "CanDrop..." vs. "Drop....".  Is there situations were the user is no allowed to drop existing connections and that's what this flag is being used to indicate?